packages/sdk-node/examples/next/package.json

```diff
@@ -1,17 +1,24 @@
 {
-  "name": "next",
-  "version": "0.2.0",
+  "name": "my-app",
```

Comment:
❓ 
Should this example be moved on the `nextjs` package or be removed entirely ?
Also if we decide to keep this, don't we need to have the `@clerk/nextjs` package in the dependencies ? _(The way it was done previously was using the locally compiled package.)_

Reply:
Yes. Moving it.